code/game/objects/structures/kitchen_spike.dm

```diff
@@ -22,58 +44,83 @@
 	var/skin = 0
 	var/skintype = null
 
+	var/global/list/allowed_types = list( //ugh I know it's a whitelist but it's not as god-awful and hacky as before. at least this way it's sustainable. i don't think there's even a better way than this.
+		/mob/living/carbon/monkey,
+		/mob/living/carbon/alien,
+		/mob/living/simple_animal/hostile/bear,
+		/mob/living/simple_animal/pet/dog/corgi,
+	)
+
+	var/global/list/meatlist = list(
+		"monkey"	= MEATTYPE_MONKEY,
+		"alien"		= MEATTYPE_ALIEN,
+		"bear"		= MEATTYPE_BEAR,
+		"corgi"		= MEATTYPE_CORGI,
+	)
+	var/global/list/skinlist = list(
+		"monkey"	= SKINTYPE_MONKEY,
+		"alien"		= SKINTYPE_ALIEN,
+		"bear"		= SKINTYPE_BEAR,
+		"corgi"		= SKINTYPE_CORGI,
+	)

```

Comment:
If you want a variable that's completely global, you just declare it with no indentation. Lists like this are what global is for.